pages/phases/phone-assessment-interviews/training.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/training/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 011
```

Comment:
I might just leave a little more wiggle room with the sub-phase-order number and make it 015. That would still put it between scheduling (010) and Interviewing applicants (020). Call me paranoid, but I suspect there may be more content in future?

Reply:
Good point. 👍